test/api/get.spec.js

```diff
@@ -0,0 +1,125 @@
+/* eslint-env mocha */
```

Comment:
Do these tests cover something that interface-ipfs-core tests don't cover? If yes, why not have all of them at interface-ipfs-core for get?